### PR TITLE
add Hjson

### PR DESCRIPTION
**Pull request by laktak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -358,7 +358,7 @@ impl serde::Serialize for i32 {
 As you can see it's pretty simple. More complex types like `BTreeMap` need to
 pass a
 [MapVisitor](http://serde-rs.github.io/serde/serde/serde/ser/trait.MapVisitor.html)
-to the
```
Comment (oli-obk) [inline README.md]: please leave those spaces in there, I know they are unfortunate, but otherwise the next line is very long.

Comment (oli-obk) [inline README.md]: ah, looking at the rendered version, this doesn't seem to be an issue

**Comment by oli-obk** [comment]
thanks! great to have more (de)serializers here.

File: README.md (hunk 1 above)
Comment (laktak) [inline README.md]: OK, sorry. That was my editor and I have `git diff` set to ignore space.

**Pull request merged**